SR2024/2023-12-13-challenge-reminder.md

```diff
@@ -0,0 +1,20 @@
+---
+to: Student Robotics 2024
+subject: Student Robotics Challenge submissions
+---
+
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+We hope you had an excellent new year and holiday break.
+
+Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline this past weekend. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.
+
+If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)
+
+Congratulations to the 15 teams that managed to complete a challenge by that deadline.
```

Comment:
This line duplicates the first sentence of the second paragraph. Is that expected? (it feels a bit redundant to me -- perhaps just drop this line?)
```suggestion
```